substrate/client/sysinfo/src/sysinfo.rs

```diff
@@ -659,9 +727,17 @@ pub fn gather_hwbench(scratch_directory: Option<&Path>) -> HwBench {
 
 impl Requirements {
 	/// Whether the hardware requirements are met by the provided benchmark results.
-	pub fn check_hardware(&self, hwbench: &HwBench) -> Result<(), CheckFailures> {
+	pub fn check_hardware(
+		&self,
+		hwbench: &HwBench,
+		is_rc_authority: bool,
+	) -> Result<(), CheckFailures> {
 		let mut failures = Vec::new();
 		for requirement in self.0.iter() {
+			if requirement.check_on_rc_authority && !is_rc_authority {
```

Comment:
Not sure if this makes sense? There can just be two hardware presents, one for authority and one for para node?

Reply:
Yes, the number of cpu cores requirement is only for validators running on the relay chain which do a lot more work than a parachain node, so I don't see why we would need to enforce the same requirements for collators.

Reply:
Yea i get this, but what i mean is here:  

https://github.com/paritytech/polkadot-sdk/blob/62f00f6ab8db987f7f440bc0c450356b9ae092bf/substrate/bin/node/cli/src/service.rs#L556

I think its cleaner to have one JSON for validators and one for parachain collators instead of putting contextual information like `check_on_rc_authority` into the JSON itself.

Reply:
Actually maybe its not such a good idea i think to have two reference hardware. The whole joke of the reference hardware is that it is fixed to something. Not that there are different versions of it.  
It would probably confuse people when the parachain binary reports something different than the polkadot binary - while both are claiming to check for "reference hardware compatibility". People are always free to ignore the results if they are too restructive.

Reply:
> I think its cleaner to have one JSON for validators and one for parachain collators instead of putting contextual information like check_on_rc_authority into the JSON itself.

Yeah, I thought about that as well, the only reason I did not go that route is because I thought is going to be duplicate numbers which can easily fall out of sync, but I guess I can make sure that doesn't happen with a unittest, I can go this route if you prefer it this way.


> Actually maybe its not such a good idea i think to have two reference hardwares. The whole joke of the reference hardware is that it is fixed to something. Not that there are different versions of it.
It would probably confuse people when the parachain binary reports something different than the polkadot binary - while both are claiming to check for "reference hardware compatibility". People are always free to ignore the results if they are too restrictive.

From my point of view having different hardware requirements between relay chain validators and parachain collators seems more a feature than a bug to me, because these are 2 binaries that are doing different jobs and more likely than not they will be run by a different set of people. 

Although on the single core performance they will have to have same requirements because of the weights, when we talk about the necessary number of cores, they will naturally evolve to have different load profiles because they run different protocols, so I don't think it would be a good idea to impose the same restrictions on collators as on validators.

Reply:
Okay then lets just go with the current approach and monitor the feedback from the validators in the lounges channels.